src/index.ts

```diff
@@ -176,21 +159,22 @@ export const parseDnsDidResults = (recordSet: IDNSRecord[] = [], dnssec: boolean
 /**
  * Queries a given domain and parses the results to retrieve openattestation document store records if any
  * @param domain e.g: "example.openattestation.com"
+ * @param customDnsResolvers
  * @example
  * > getDocumentStoreRecords("example.openattestation.com")
  * > [ { type: 'openatts',
-    net: 'ethereum',
-    netId: '3',
-    addr: '0x2f60375e8144e16Adf1979936301D8341D58C36C',
-    dnssec: true } ]
+ net: 'ethereum',
+ netId: '3',
+ addr: '0x2f60375e8144e16Adf1979936301D8341D58C36C',
+ dnssec: true } ]
  */
 export const getDocumentStoreRecords = async (
   domain: string,
   customDnsResolvers?: CustomDnsResolver[]
 ): Promise<OpenAttestationDNSTextRecord[]> => {
   trace(`Received request to resolve ${domain}`);
 
-  const dnsResolvers = customDnsResolvers || defaultDnsResolvers;
+  const dnsResolvers = customDnsResolvers ?? defaultDnsResolvers;
```

Comment:
```suggestion
  const dnsResolvers = customDnsResolvers || defaultDnsResolvers;
```
I believe the behaviour we want here is still the OR operator `||`?

If a falsy value happens to be provided erroneously on runtime/JavaScript:

```javascript
✅ console.log(0     || "fallback") // "fallback"
✅ console.log(""    || "fallback") // "fallback"
✅ console.log(false || "fallback") // "fallback"

❌ console.log(0     ?? "fallback") // 0
❌ console.log(""    ?? "fallback") // ""
❌ console.log(false ?? "fallback") // false
```

Reply:
Also can. I was thinking, in our case, `customDnsResolvers` is optional and typed as `CustomDnsResolver[]`. So it can only be `undefined` or  an array of `CustomDnsResolver`, and wouldn’t be a falsey value, would it?